hieradata_aws/production.yaml

```diff
@@ -193,8 +193,6 @@ govuk::apps::travel_advice_publisher::enable_email_alerts: true
 govuk_search::monitoring::es_port: '80'
 govuk_search::monitoring::es_host: 'vpc-blue-elasticsearch6-domain-2hnib7uydv3tgebhabx74gdelq.eu-west-1.es.amazonaws.com'
 
-govuk_jenkins::jobs::enhanced_ecommerce_search_api::cron_schedule: '0 6 * * 1-5' # Every weekday at 6am
```

Comment:
Is https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_jenkins/templates/jobs/enhanced_ecommerce.yaml.erb going to work with this being `undef`?

Reply:
Maybe not - I've put in a fix for that